.github/workflows/cr.qa.deploy.yaml

```diff
@@ -0,0 +1,74 @@
+name: Deploy to Amazon ECS (QA)
+
+on:
+  workflow_dispatch: # Manually triggered from GitHub Actions tab
+  pull_request:
+    types: [labeled] # Triggered by a label in a PR
+
+env:
+  AWS_REGION: us-east-1
+  ECR_REPOSITORY: bim-dao-frontend-qa
+  ECS_SERVICE: bim-dao-frontend
+  ECS_CLUSTER: bim-dao-frontend-qa
+  ECS_TASK_DEFINITION: bim-dao-frontend-qa
+  CONTAINER_NAME: bim-dao-frontend
+
+  PROFILE: testnet.qa
```

Comment:
This is not going to fly. https://github.com/RootstockCollective/dao-frontend/blob/develop/Dockerfile#L27 selects the right `.env.*` file for the environment, but it isn't handling `testnet.qa`.

Reply:
done